### ISSUE TITLE
MStore Pro Build Failed on Xcode 

### ISSUE THREAD

**Issue by Jegufull** [inspireui/support#389]
**Required Info (please complete the following information):**
 - Product: [e.g. MStore, ListApp]
 - Your Purchased Code: [get from https://codecanyon.net/downloads]
(we may not reply if you are not our customer)

**Describe the bug**

**Screenshots**

**Additional context**
Add any other detail logs about the problem here.
If possible please share your config.js that use for the app setting

**Comment by Jegufull** [comment]
Product: [MStore-Pro]
Purchased Code: [36e9cfd7-96da-413e-b43f-83ef03bed15b]

Hello, I get the following error. I reinstalled the libraries and the error continues, what could I do?

'FBSDKCoreKit/FBSDKCoreKit.h' file not found

I reinstalled the libraries and the error continues, what could I do?

![Captura de pantalla 2019-03-11 a la(s) 12 05 38 p  m](https://user-images.githubusercontent.com/47310507/54142826-855d2280-43f6-11e9-8db2-d415bb640c63.png)
